<launch>
        <param name="robot_description" command="cat $(find TonyPi)/model.xml" />
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="state_publisher" pkg="TonyPi" type="state_publisher" />
</launch>
